using-the-rest-api/client-libraries.md

```diff
@@ -22,3 +22,6 @@ The [Backbone.js client](https://developer.wordpress.org/rest-api/backbone-javas
 
 ## C# / .NET
 [WordPressPCL](https://github.com/wp-net/WordPressPCL): a full REST API client written in C#.
+
+## Dart / Flutter
+[wordpress_client](https://pub.dev/packages/wordpress_client): A library exposing a fluent api to interact with the wordpress REST API.
```

Comment:
The API design here doesn't quite match what I think of when I read [fluent](https://en.wikipedia.org/wiki/Fluent_interface), but this is a good addition -- thank you!

Reply:
Thanks a bunch!